config/config.h

```diff
@@ -21,23 +23,13 @@ enum ConfigOption {
   QAT_PERIODICAL_POLLING,
   QAT_COMPRESSION_LEVEL,
   LOG_LEVEL,
-  LOG_STATS_SAMPLES
+  LOG_STATS_SAMPLES,
+  CONFIG_MAX
 };
 
-extern int use_qat_compress;
-extern int use_qat_uncompress;
-extern int use_iaa_compress;
-extern int use_iaa_uncompress;
-extern int use_zlib_compress;
-extern int use_zlib_uncompress;
-extern int iaa_compress_percentage;
-extern int iaa_uncompress_percentage;
-extern int iaa_prepend_empty_block;
-extern int qat_periodical_polling;
-extern int qat_compression_level;
 extern std::string log_file;
-extern int log_level;
-extern int log_stats_samples;
+
+extern uint16_t configs[CONFIG_MAX];
```

Comment:
Let's make it int32_t. For RocksDB testing, for example, I had log_stats_samples set to 1M (it would not fit in 16 bits). Also, signed integers would allow options with negative values, if needed in the future.

Reply:
I see you changed it to uint32_t. Leaving it unsigned is fine for now. If we ever need to change to signed, we can. It's an internal implementation detail. It doesn't change the API (config file) exposed to the user.